src/exchanges.rs

```diff
@@ -156,6 +157,11 @@ async fn exchange_rate_getter<Fut>(
         };
 
         log::info!("New exchange rate polled: {:?}", raw_rate);
+        if let Some(ref mut conn) = database_conn {
+            if let Err(e) = crate::database::write_read_rate(conn, raw_rate) {
+                log::error!("Unable to INSERT new reading: {}, due to: {}", raw_rate, e)
+            };
+        }
```

Comment:
Ideally these errors would also be logged in prometheus so that they can be more easily monitored.